index.html

```diff
@@ -19,23 +19,170 @@
-            <p class="background">
-                I can help you build a product , feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded, do not hestiate to contact me.
-            </p>
-        </div>
-        <ul class="social-media  container">
-            <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
-            <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
-            <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
-            <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
-            <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
+    <div class="container">
+        <h1 class="logo-message">I'm  Dan</h1>
+        <h3 class="dev-intro">I'am a Software Developer</h3>
+        <p class="background">
+            I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, do not hestiate to contact me.
+        </p>
+    </div>
+    <ul class="social-media  container">
+        <li><a href="https://github.com/dunnyk"><img src="image/github.png" alt="github icon"></a></li>
+        <li><a href="https://www.linkedin.com/in/daniel-njoroge-8091b3128/"><img src="image/linkedin.png" alt="linkedin icon"></a></li>
+        <li><a href="https://twitter.com/dankama231"><img src="image/twitter.png" alt="twitter icon"></a></li>
+        <li><a href="https://www.instagram.com/danielkamau231/"><img src="image/finger.png" alt="peace finger sign"></a></li>
+        <li><a href="https://medium.com/@danielkamau231"><img src="image/medium.png" alt="M logo"></a></li>
+    </ul>
+  </section>
+
+
+  <!-- WORK SECTION -->
+
+  <section class="main-wrap">
+    <article class="work-section">
+      <div class="work-title">
+        <h1>My Recent Works</h1><br>
+        <hr class="line"/>
+      </div>
+        <img src="image/Tonic.png" alt="Tonic Image">
+      <div class="multi-post">
+        <h3>Multi-Post Stories</h3>
+        <p>
+         A daily selection of privately personalized reads; no account or sigh-ups
+         required has been the industry's standard dummy
+         text ever since the 1500s, when unkown printer took a standard dummy text.
+        </p>
+      </div>
+      <div class="work-languages">
+        <ul class="work-list">
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
         </ul>
-    </section>
+      </div>
+      <button type="button" class="btn">See Project</button>
+    </article>
+
+  <!-- CARDS -->
+<section class="grid-container">
```

Comment:
- [ ] Kindly replace the section tag for each project card with a div tag, this is more semantic as all projects belong to this section 😉 
- [ ] Kindly align the ul tags to be like the Figma design for each card, please see the screenshot above
- [ ] Currently I see a dark background by the right pleaase check screenshot below, it would be better to remove this and also make the background color same as the figma design 🤝 


 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_12](https://user-images.githubusercontent.com/69638013/160237103-a84105ad-0a94-49aa-bc19-bff7cc4d988e.png)  |  ![Screenshot_13](https://user-images.githubusercontent.com/69638013/160237098-683595e4-6c27-4093-b5b6-28eb7539a152.png)